src/main/kotlin/org/jitsi/nlj/util/SsrcAssociation.kt

```diff
@@ -1,11 +1,39 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package org.jitsi.nlj.util
 
 import org.jitsi.nlj.rtp.SsrcAssociationType
 
-data class SsrcAssociation(
+abstract class SsrcAssociation(
     val primarySsrc: Long,
     val secondarySsrc: Long,
     val type: SsrcAssociationType
 ) {
     override fun toString(): String = "$secondarySsrc -> $primarySsrc ($type)"
 }
+
+class LocalSsrcAssociation(
```

Comment:
Nit: can we document what these are? Are they "local" as in "belonging to the bridge" (like we have for "audio ssrc" and "video ssrc"), or as in "belonging to this endpoint as opposed to another endpoint".

Reply:
yup, i'll add some docs for that